tests/features/search/querying.feature

```diff
@@ -5,7 +5,8 @@ Feature: Search querying
     When I visit a `search page`
     And I enter "paris" in the `search box`
     And I click the `search button`
-    Then I see a `search result`
+    Then I see "paris" in the `search box`
```

Comment:
This is always failing for me when using the GeckoDriver:
```
Then I see "paris" in the `search box` # tests/features/support/step-definitions.js:44
    Error while running "getValue" command
    + expected - actual
 
    +paris

    AssertionError: Error while running "getValue" command: expected '' to equal 'paris'
        at NightwatchAPI.client.getValue (/incubator-portal-vue-nuxt/tests/features/support/step-runners.js:124:44)
        at transport.locateElement.then.then.catch.then.result (/incubator-portal-vue-nuxt/node_modules/nightwatch/lib/api-loader/element-command.js:144:30)
        at processTicksAndRejections (internal/process/next_tick.js:81:5)
```

Does it pass for you?
First I thought it might be a timing issue, but even a 4 second timeout doesn't help. Although I can see the text in the field while the test runs, so maybe something with the web driver not inspecting it correctly?

I'm on FF-67

Reply:
As discussed elsewhere, given the various issues we are experiencing with geckodriver, we will instead stop using it in `test:e2e` but leave it set up in the project should the problems be resolved upstream.